akka-stream/src/main/resources/reference.conf

```diff
@@ -42,5 +42,10 @@ akka {
       output-burst-limit = 1000
     }
 
+    # Fully qualified config path which holds the dispatcher configuration
+    # to be used by FlowMaterialiser when creating Actors for IO operations,
+    # such as FileSource, FileSink and others.
+    file-io-dispatcher = ${akka.io.tcp.file-io-dispatcher} # TODO better provide a thread pool backed one here by default?
+                                                           # note: tcps dispatcher is the is the akka.actor.default-dispatcher
```

Comment:
TODO: I'm working on the pre-configured dispatcher for this guy here.
I ran into trouble when wanting to provide a default special dispatcher for these, and still allow users to use `.withAttributes(dispatcher...)`, I'll update the PR soon today.